### ISSUE TITLE
Firefox and Adobe Reader DC problem

### ISSUE THREAD

**Issue by mina354** [Skulltrail192/One-Core-API-Binaries#64]
Firefox hangs if you leave it idle for sometime, then crashes with these 2 errors appearing. This happens with version 52.9.0esr (only if One-Core-API is installed, it doesn't happen when One-Core-API isn't installed)
![image](https://user-images.githubusercontent.com/87821255/172023245-c29dd448-5530-47df-ae83-ec20ad8ec098.png)
And it crashes often with these appearing. This happens with version 53 and 54 only. 53 and 54 also have the same issue too as 52.9.0esr (the issue wrotten above)
![image](https://user-images.githubusercontent.com/87821255/172023463-9591926a-3048-41fc-b8d6-fbd95c8635bb.png)
Also, everytime I open Adobe Reader DC, I get this error and I am not able to use Adobe Reader DC because of it.
![image](https://user-images.githubusercontent.com/87821255/172024456-51908db8-4b3e-48cf-9ae2-4cd291b03aa7.png)
I found a solution to this, but it's a bad one, because it's about deleting all the plug-ins, which renders the application useless, but doesn't make the error appear at the same time. I've been trying to look for a better solution to this for a long time, but couldn't.

**Comment by Skulltrail192** [comment]
This is a very strange. I always test firefox 52 ESR and firefox works until version 54. What environment? Can you post the link to iso utilized?

**Comment by mina354** [comment]
Environment: VMware Workstation Pro 16.2.3
Link to ISO used: [here](http://jerryching.spdns.de/Software/ISO%20Backup/Windows%20XP/GRTMPVOL_EN.iso)

**Comment by mina354** [comment]
> > Firefox hangs if you leave it idle for sometime, then crashes with these 2 errors appearing. This happens with version 52.9.0esr (only if One-Core-API is installed, it doesn't happen when One-Core-API isn't installed) ![image](https://user-images.githubusercontent.com/87821255/172023245-c29dd448-5530-47df-ae83-ec20ad8ec098.png) And it crashes often with these appearing. This happens with version 53 and 54 only. 53 and 54 also have the same issue too as 52.9.0esr (the issue wrotten above) ![image](https://user-images.githubusercontent.com/87821255/172023463-9591926a-3048-41fc-b8d6-fbd95c8635bb.png) Also, everytime I open Adobe Reader DC, I get this error and I am not able to use Adobe Reader DC because of it. ![image](https://user-images.githubusercontent.com/87821255/172024456-51908db8-4b3e-48cf-9ae2-4cd291b03aa7.png) I found a solution to this, but it's a bad one, because it's about deleting all the plug-ins, which renders the application useless, but doesn't make the error appear at the same time. I've been trying to look for a better solution to this for a long time, but couldn't. And weirdly enough, during the Adobe Reader DC error pop-up, some random Chinese folder becomes created on the desktop: ![image](https://user-images.githubusercontent.com/87821255/172025435-a5e0688d-7fe5-43e2-b7a2-c1b4604a916c.png)
> 
> Well, all packages are installed?

Yes, sir.

**Comment by Skulltrail192** [comment]
So this iso is Windows XP SP3 RTM without updates?

**Comment by mina354** [comment]
> So this iso is Windows XP SP3 RTM without updates?

Yes.

**Comment by mina354** [comment]
@Skulltrail192 So wait, do I need a fully updated XP SP3 to install One-Core-API? Or it's ok without updates?

**Comment by Skulltrail192** [comment]
> @Skulltrail192 So wait, do I need a fully updated XP SP3 to install One-Core-API? Or it's ok without updates?

No, i tested with your iso and it is working fine.

**Comment by mina354** [comment]
Oh, well. Hope you started investigation of Adobe Reader's issue.

**Comment by Skulltrail192** [comment]
Well, i isoleted the plugins folder and i'm investiganting. It is a issue introduced on Acobe reader DC 2021+. I was tested with Adobe Reader DC 2017.

**Comment by mina354** [comment]
Thanks for updating me up.

**Comment by Skulltrail192** [comment]
The issue related with Acrobat Reader DC seem installation problem. When i copied the installation folder from Windows 7, i get this error. However, if i extract the installer and edit msi with **orca**, it install and work without this error. However, i see a memory leak with RdrCEF.exe, and i investigating it.

**Comment by mina354** [comment]
Yes, I did edit the msi first, then install it. At that time I posted this issue (June 4), I had only Windows Installer 4.5 installed.

**Comment by Skulltrail192** [comment]
Well, i downloaded latest Acrobat Reader for Windows 7 32 bits and edited msi installer. After installation, i't loads fine. So, maybe new version of one-core-api maybe solved or your version of acrobat reader is decreapted. The problem for now is RdrCEF.exe.

**Comment by mina354** [comment]
Well, when the new version of One-Core-API comes out, I will test latest Adobe Acrobat Reader DC and tell you if it's solved or not.

**Comment by mina354** [comment]
Is this issue not solved yet? I really hope this gets fixed either on version 2.9.0, or in the future.

**Comment by Skulltrail192** [comment]
Sorry man, but. not, i is not solved. I'm only one man and have several bugs to solve.

**Comment by mina354** [comment]
> Sorry man, but. not, i is not solved. I'm only one man and have several bugs to solve.

It's fine, I can wait.

**Comment by WindowsCola** [comment]
for adobe reader,try to delete C:\Program Files\Adobe\Acrobat Reader DC\Reader\plug_ins and it should work 
<img width="1046" alt="Screenshot 2022-09-10 at 1 05 44 PM" src="https://user-images.githubusercontent.com/113223323/189470013-6be1a860-508f-4b1e-a269-6b5bb34163b8.png">

**Comment by Smu1zel** [comment]
> for adobe reader,try to delete C:\Program Files\Adobe\Acrobat Reader DC\Reader\plug_ins and it should work <img alt="Screenshot 2022-09-10 at 1 05 44 PM" width="1046" src="https://user-images.githubusercontent.com/113223323/189470013-6be1a860-508f-4b1e-a269-6b5bb34163b8.png">

I like how we can run this on XP now, while Wine can't even get past the installer.

**Comment by mina354** [comment]
> for adobe reader,try to delete C:\Program Files\Adobe\Acrobat Reader DC\Reader\plug_ins and it should work <img alt="Screenshot 2022-09-10 at 1 05 44 PM" width="1046" src="https://user-images.githubusercontent.com/113223323/189470013-6be1a860-508f-4b1e-a269-6b5bb34163b8.png">

Yes, but it renders the program useless then.

**Comment by WindowsCola** [comment]
> > for adobe reader,try to delete C:\Program Files\Adobe\Acrobat Reader DC\Reader\plug_ins and it should work <img alt="Screenshot 2022-09-10 at 1 05 44 PM" width="1046" src="https://user-images.githubusercontent.com/113223323/189470013-6be1a860-508f-4b1e-a269-6b5bb34163b8.png">
> 
> Yes, but it renders the program useless then.

Yeah even the adobe reader itself has a lot of limation on using the tools.The only purpose of using this app is to use it as a pdf reader,simple signing and add words on the pdf.

**Comment by WindowsCola** [comment]
> > for adobe reader,try to delete C:\Program Files\Adobe\Acrobat Reader DC\Reader\plug_ins and it should work <img alt="Screenshot 2022-09-10 at 1 05 44 PM" width="1046" src="https://user-images.githubusercontent.com/113223323/189470013-6be1a860-508f-4b1e-a269-6b5bb34163b8.png">
> 
> I like how we can run this on XP now, while Wine can't even get past the installer.

you need to download the offline installer and install the orca app https://www.technipages.com/downloads/OrcaMSI.zip ,then extract thesetup file and right the file that ended with .msi and click the option with 'edit with orca'.Then find the 'LanuchCondition' option and delete all the strings there.Finally save the files and it should work.Besides after you install the file, there should be a file edned with .msp,double click it and install the latest patches of Adobe Reader and you are done.

**Comment by Skulltrail192** [comment]
That was fixed on version 4.0.0.
